pennylane_qiskit/devices.py

```diff
@@ -137,6 +152,11 @@ def operations(self):
     def expectations(self):
         return set(self._expectations)
 
+    @property
+    def backend(self):
+        # type: () -> BaseBackend
+        return self._provider.get_backend(self.backend_name)
+
     def apply(self, operation, wires, par):
```

Comment:
Some operations might not have parameters. Better to perhaps put `par = None` here?

Reply:
This is an overwritten abstract method of `Device`, so the signature will need to match :)

Reply:
See https://github.com/XanaduAI/pennylane/blob/478e0fa6fd692768105002d8a09d99e70ae7644b/pennylane/_device.py#L321 (cf. [Liskov substitution principle](https://en.wikipedia.org/wiki/Liskov_substitution_principle))